addons/armaos/functions/fnc_os_cd.sqf

```diff
@@ -1,14 +1,26 @@
-params ["_options", "_consoleInput"];
+/**
+ * Changes/sets the current working directory of a given terminal on a given computer.
+ * Also returns informations about the success of the command.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Folder/Directory <[STRING]>
+ *
+ * Results:
+ * 1: Informations <[STRING]>
+ */
+
+params ["_computer", "_options"];
 
 _options = _options joinString " ";
```

Comment:
private

Reply:
This is one of the function parameters, that is converted from array to joined string. I think the private statement isn't needed in this case. Or Am I wrong?

Reply:
Your right